src/modules/todo.js

```diff
@@ -0,0 +1,101 @@
+const taskContainer = document.querySelector('.task-container');
+
+export default class myTodo {
+  constructor() {
+    this.tasks = [];
+  }
+
+    addTask = (description) => {
+      const myTask = {
+        description,
+        completed: false,
+        index: this.tasks.length + 1,
+      };
+      this.tasks.push(myTask);
+      this.setStorage();
+    }
+
+    remove = (id) => {
+      this.resetIndex();
+      this.tasks.forEach((task) => {
+        if (task.index === Number(id)) {
+          this.tasks
+            .splice(task.index - 1, 1);
+        }
+      });
+    }
+
+    resetIndex = () => {
+      let initialIndex = 1;
+      this.tasks.forEach((task) => {
+        task.index = initialIndex;
+        initialIndex += 1;
+      });
+    }
+
+    cleanCompleted = () => {
+      this.tasks = this.tasks.filter((task) => task.completed === false);
+    }
+
+    taskTemplate = (task) => `
+        <div class="task">
+            <div class="t-left" id=${task.index}>
+                <input type="checkbox" name="Drag" class='checkbox' ${task.completed ? 'checked' : ''} id=${task.index}> 
+                <label for="" class=${task.completed ? 'line task-label' : 'task-label'}>${task.description}</label>
+            </div>
+            <p class="deleteTask" id=${task.index}>Delete</p> 
+        </div>`;
+
+    displayTasks = () => {
+      this.getStorage();
+      taskContainer.innerHTML = '';
+      this.tasks
+        .forEach((task) => taskContainer.insertAdjacentHTML('beforeend', this.taskTemplate(task)));
+      const checkboxes = document.querySelectorAll('.checkbox');
+      const taskLabel = taskContainer.querySelectorAll('.task-label');
+      checkboxes.forEach((checkbox, id) => {
+        checkbox.addEventListener('change', () => {
+          if (checkbox.checked) {
+            taskLabel[id]?.classList.add('line');
+            this.tasks = this.tasks.map((task) => {
+              if (task.index === parseInt(checkbox.parentElement.id, 10)) {
```

Comment:
- [ ] Great job by ensuring that property index is set to the value of the new array length, however, remaining task indexes are not updated after deleting as a task. Kindly ensure that when a task is deleted all indexes of remaining items are updated accordingly, for instance, if first item is deleted, then second item should have it's index updated from `2` to `1` and the third item from `3` to `2` and so on.